### ISSUE TITLE
Tab Length doesn't seem to work?

### ISSUE THREAD

**Issue by chrisdrackett** [atom/language-gfm#134]
I have a Tab Lentgh in the settings set to 2 spaces, but when I tab in a file I always get 4. not sure if I'm missing something...

**Comment by lee-dohm** [comment]
Because things in Markdown are based on multiples of four spaces, the default in Markdown is 4 spaces regardless of your global setting. You can change it by going to the language-gfm settings in the Settings View:

1. Launch Atom
1. Open the Settings View with <kbd>Cmd+,</kbd> on OS X or <kbd>Ctrl+,</kbd> on other platforms
1. Click the Packages tab
1. Search for `language-gfm`
1. Click the Settings button on the language-gfm card
1. Scroll down to see "Tab Length"
1. Set it to whatever value you prefer

**Comment by chrisdrackett** [comment]
Right, this is what I'm talking about, this is where I have it set to 2 spaces

**Comment by lee-dohm** [comment]
I can't reproduce the behavior you're describing using Atom v1.5.0-dev-f3e257c on Mac OS X 10.11.1. Have you tried closing and opening the Markdown document after you change the setting?

Have you followed the suggestions in the [Debugging document](https://atom.io/docs/latest/hacking-atom-debugging)? Also, could you give the following information?

* Which version of Atom are you using?
* What OS and version are you running?
* Do you have [Emmet](https://atom.io/packages/emmet) installed?
* Can you reproduce the issue after completely exiting Atom and restarting using `atom --safe`?
